src/test_config.h

```diff
@@ -0,0 +1,6 @@
+// Filled in by the build system
+
+#pragma once
```

Comment:
this file should not be commited

Reply:
This file appeared automatically somehow.

Reply:
I see why.

Reply:
File removed.